integration_test/cases/repo.exs

```diff
@@ -632,6 +633,7 @@ defmodule Ecto.Integration.RepoTest do
     assert TestRepo.aggregate(query, :count, :visits) == 3
   end
 
+  @tag :insert_cell_wise_defaults
```

Comment:
Sure. In Postgres, you can write `INSERT` statements that use `DEFAULT` value for individual column values. If you are doing a multi-row insert, different columns can be defaulted for different rows.

This is not possible in SQLite. As [SQLite's documentation for `INSERT`](https://www.sqlite.org/lang_insert.html) shows, you can _either_ have a global `DEFAULT VALUES` clause or you can explicitly provide values. You can not use `DEFAULT` inside the `VALUES(...)` clause.